src/AspNet.Security.OAuth.Apple/AppleAuthenticationHandler.cs

```diff
@@ -283,19 +275,19 @@ protected override async Task<HandleRequestResult> HandleRemoteAuthenticateAsync
 
         if (Options.SaveTokens)
         {
-            var authTokens = new List<AuthenticationToken>()
-            {
-                new AuthenticationToken() { Name = "access_token", Value = tokens.AccessToken },
-            };
+            List<AuthenticationToken> authTokens =
```

Comment:
Assuming we like the new syntax mostly for its terseness, maybe we should put everything on the same line when there's a single item added?

```csharp
List<AuthenticationToken> authTokens = [new() { Name = "access_token", Value = tokens.AccessToken }];

Reply:
I'm not going to change this one, at least for now, as doing so needs me to suppress SA1010 due to [this bug](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3687).